src/main.server.ts

```diff
@@ -1,7 +1,33 @@
 import { bootstrapApplication } from "@angular/platform-browser";
 import { AppComponent } from "./app/app.component";
 import { config } from "./app/app.config.server";
+import * as Sentry from "@sentry/angular";
 
+// @ts-expect-error i want this to compile, but it doesn't
+const SENTRY_DSN = import.meta?.env?.NG_APP_SENTRY_DSN;
+// @ts-expect-error i want this to compile, but it doesn't
+const SENTRY_TRACE_SAMPLE_RATE = import.meta?.env?.NG_APP_SENTRY_TRACE_SAMPLE_RATE || '0';
+// @ts-expect-error i want this to compile, but it doesn't
+const SENTRY_PROFILE_SAMPLE_RATE = import.meta?.env?.NG_APP_SENTRY_PROFILE_SAMPLE_RATE || '0';
+// @ts-expect-error i want this to compile, but it doesn't
+const SENTRY_ENVIRONMENT = import.meta?.env?.NG_APP_SENTRY_ENVIRONMENT || 'development';
+// @ts-expect-error i want this to compile, but it doesn't
+const SENTRY_RELEASE = import.meta?.env?.NG_APP_SENTRY_RELEASE || undefined;
+
+const BACKEND_BASE_URI = import.meta.env['NG_APP_BACKEND_BASE_URI'];
+
+if(SENTRY_DSN && SENTRY_DSN !== '') {
+  Sentry.init({
+    dsn: SENTRY_DSN,
+    environment: SENTRY_ENVIRONMENT + 'server',
```

Comment:
I think this is still not OK :(

The environment is dev/staging/prod etc, which is very different from differentiating between client-side rendering VS server-side rendering. Let's use a custom context or tag for SSR?

Reply:
probably tagging so that we can search, e.g. `rendered_at`: `client` or `server`.

Reply:
I would be better to use diffierent DSNs for server and client side data anyways. This is just leftover code.